pkg/pbq/store/wal/metrics.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wal
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	LabelPartitionKey = "partitionKey"
+)
+
+// TODO - Adjust metric bucket range after we get more map reduce use cases.
+
+var entriesCount = promauto.NewCounterVec(prometheus.CounterOpts{
+	Subsystem: "pbq_wal",
+	Name:      "wal_entries_total",
+	Help:      "Total number of entries written across ALL wal files/partitions",
+}, []string{})
```

Comment:
If there's no labels, why don't use `Counter`?

Reply:
Good question, I leave the label list empty because currently WAL object doesn't have attributes about pipeline name and vertex name etc. So when creating such metric, we are creating a global count.

I think we should add labels to separate WAL metrics per pipeline/vertex etc. Thinking about addressing it as a separate issue.

Reply:
https://github.com/numaproj/numaflow/issues/365